project-management/src/test/groovy/life/qbic/projectmanagement/application/ExperimentInformationServiceSpec.groovy

```diff
@@ -111,14 +111,14 @@ class ExperimentInformationServiceSpec extends Specification {
         experiment.addVariableToDesign(experimentalVariable.name().value(), experimentalVariable.levels().collect { it.experimentalValue() })
 
         when: "experimental groups are added to an experiment"
-        def group1 = new ExperimentalGroupDTO(List.of(experimentalVariable.levels().get(0)), 5)
-        def group2 = new ExperimentalGroupDTO(List.of(experimentalVariable.levels().get(1)), 6)
+        def group1 = new ExperimentalGroupDTO("label1", List.of(experimentalVariable.levels().get(0)), 5)
```

Comment:
NIT: I'd keep the naming consistent throughout the entire application, so it's either a label or a name